container/docker/engine.py

```diff
@@ -780,3 +726,60 @@ def print_version_info(self):
         client = self.get_client()
         pprint.pprint(client.info())
         pprint.pprint(client.version())
+
+    def bootstrap_env(self, temp_dir, behavior, operation, compose_option,
+                      builder_img_id=None, context=None):
+        """
+        Builds the environment for common operations for compose engine,
+        like orchestration, termination, etc.
```

Comment:
This sentence doesn't quite make sense. Should be something like: 

Build common Docker Compose elements required to execute orchestrate, terminate, restart, etc.